src/modules/site-v2/base/utils/tools.py

```diff
@@ -79,19 +79,14 @@ def try_submit(kind, user, data, no_cache):
     job = get_pipeline_class(kind=kind).create(user, data, no_cache=no_cache, valid_file_extensions=TOOL_INPUT_DATA_VALID_FILE_EXTENSIONS)
 
     # Schedule the job, if applicable
-    #   CachedDataError    -> the job already has results
+    #   JobAlreadyScheduledError -> a job computing these results has already been scheduled
```

Comment:
Part of the new explicit separation between job creation & submission. This error name is more accurate in the new flow, and is a bit more general (catches a job that's already been submitted & redirects to that ongoing submission)